extension/content/components/recipes/details/DetailsHeader.js

```diff
@@ -127,6 +134,14 @@ export default function DetailsHeader() {
         {viewExperimentButton}
         {requestApprovalButton}
         {statusToggleButton}
+        <IconButton
+          appearance="primary"
+          className="ml-1"
+          icon={<Icon icon="copy" />}
+          onClick={handleCopyClick}
+        >
+          Copy Recipe
+        </IconButton>
```

Comment:
The header is already getting pretty full in "half-width" view. I think a better solution for features like this that are less "everyday" use is to put a new button with an `ellipsis-h` icon at the end of the header. It should not be a primary button and clicking it should give you a dropdown with a submenu of tasks. Somewhat similar to the dropdown you get when you click on your avatar in the top right corner.